### DISCUSSION TITLE
What way do you recommend to copy a design to another blog?

### DISCUSSION THREAD

**Discussion by 3muchfun** [GetPublii/Publii#1551] [Help]
Assuming I have 2 blog with different posts, how do I copy a complete design from one blog to another?
I customized a blog with the Mercury theme (CSS in the App under Tools & Plugins, config.json changes, override folder, several settings in the App) and now I am looking for the easiest way to apply all these changes to my 2nd blog....

**Comment by candidexmedia** [comment]
It'll depend on the base theme and CSS mods, but you should be okay to just copy-paste those Custom CSS changes.

**Reply by 3muchfun** [reply]
as mentioned, I use Mercury (2700). So, if I copy the CSS styles, what about the settings within the App I made to Mercury?

**Reply by candidexmedia** [reply]
Copy the `input > themes` folder from one site to the other to transfer the theme overrides. 

I assume Site Settings and Theme Settings are saved in a json file in `input > config`. Inspect the files there, copy over what makes sense, and modify `sites.config.json` for site # 2 with the settings you want from # 1.

Just make sure to exclude db.sqlite, which is where the posts live.

**Reply by 3muchfun** [reply]
Thanks, @candidexmedia , for your effort, but that's too time consuming for me. I I think I will either clone site A und populate the clone with the posts from site B manually, or transfer each setting from site A to site B.
